templates/cases/case_read.haml

```diff
@@ -66,6 +66,8 @@
 
     .row
       .col-md-8
+        <cp-notifications></cp-notifications>
```

Comment:
Bonus points to anyone who wants to replace all the `UtilsService.displayAlert(..)` calls with something like this